tests/main/fips/task.yaml

```diff
@@ -40,6 +40,9 @@ execute: |
                         # 00007fc160532000     96K r---- /usr/lib/x86_64-linux-gnu/ossl-modules-3/fips.so
                         MATCH ' /usr/lib/.*/ossl-modules-3/fips.so' < snapd-map.out
                     fi
+
+                    # not executing anything from the snapd snap
+                    MATCH ' /snap/snapd' < snapd-map.out
```

Comment:
Hmm, how is the MATCH checking that?

Reply:
hmm, but how does MATCH implements the idea?

I was expecting A NOMATCH instead.